make-release.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+# Release process automation script. 
+# Used to create branch/tag, update VERSION files 
+# and and trigger release by force pushing changes to the release branch 
+
+# set to 1 to actually trigger changes in the release branch
+TRIGGER_RELEASE=0 
+NOCOMMIT=0
+
+while [[ "$#" -gt 0 ]]; do
+  case $1 in
+    '-t'|'--trigger-release') TRIGGER_RELEASE=1; NOCOMMIT=0; shift 0;;
+    '-v'|'--version') VERSION="$2"; shift 1;;
+    '-n'|'--no-commit') NOCOMMIT=1; TRIGGER_RELEASE=0; shift 0;;
+  esac
+  shift 1
+done
+
+bump_version () {
+  CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+
+  NEXT_VERSION=$1
+  BUMP_BRANCH=$2
+
+  git checkout "${BUMP_BRANCH}"
+
+  echo "Updating project version to ${NEXT_VERSION}"
+  npm --no-git-tag-version version "${NEXT_VERSION}"
+
+  if [[ ${NOCOMMIT} -eq 0 ]]; then
+    COMMIT_MSG="[release] Bump to ${NEXT_VERSION} in ${BUMP_BRANCH}"
+    git commit -asm "${COMMIT_MSG}"
+    git pull origin "${BUMP_BRANCH}"
+
+    PUSH_TRY="$(git push origin "${BUMP_BRANCH}")"
+    # shellcheck disable=SC2181
+    if [[ $? -gt 0 ]] || [[ $PUSH_TRY == *"protected branch hook declined"* ]]; then
+      PR_BRANCH=pr-${BUMP_BRANCH}-to-${NEXT_VERSION}
+      # create pull request for master branch, as branch is restricted
+      git branch "${PR_BRANCH}"
+      git checkout "${PR_BRANCH}"
+      git pull origin "${PR_BRANCH}"
+      git push origin "${PR_BRANCH}"
+      lastCommitComment="$(git log -1 --pretty=%B)"
+      hub pull-request -f -m "${lastCommitComment}" -b "${BUMP_BRANCH}" -h "${PR_BRANCH}"
+    fi 
+  fi
+  git checkout "${CURRENT_BRANCH}"
+}
+
+usage ()
+{
+  echo "Usage: $0 --version [VERSION TO RELEASE] [--trigger-release]"
+  echo "Example: $0 --version 7.7.0 --trigger-release"; echo
+}
+
+if [[ ! ${VERSION} ]]; then
+  usage
+  exit 1
+fi
+
+QUAY_REPO="quay.io/eclipse/che-dashboard:${VERSION}-react"
```

Comment:
My opinion: ```QUAY_REPO="quay.io/eclipse/che-dashboard:react-${VERSION}"``` will be better. WDYT?

Reply:
It's been already discussed [here](https://github.com/eclipse/che/issues/17647).

Reply:
ok